src/Services/ConvertToQtiService.php

```diff
@@ -192,20 +221,159 @@ private function convertAssessmentItem($json)
      *
      * @param array $manifest
      */
-    private function flushJobManifest(array $manifest)
+    private function flushJobManifest(Manifest $manifestContent, array $results)
     {
-        if ($this->dryRun) {
-            return;
+        $manifestFileBasename = static::MANIFEST_FILE_NAME;
+        $imsManifestXml = new DOMDocument("1.0", "UTF-8");
+        $imsManifestXml->formatOutput = true;
+        $element = $imsManifestXml->createElement("manifest");
+        $element->setAttribute("identifier", $manifestContent->getIdentifier());
+        $imsManifestXml->appendChild($element);
+
+        $manifestMetaData = $this->addManifestMetadata($manifestContent, $imsManifestXml);
+        $element->appendChild($manifestMetaData);
+
+        $organization = $this->addOrganizationInfoInManifest($manifestContent, $imsManifestXml);
+        $element->appendChild($organization);
+
+        $resourceInfo = $this->addResourceInfoInManifest($manifestContent, $imsManifestXml, $results);
+        $element->appendChild($resourceInfo);
+
+        $this->decorateImsManifestRootElement($element);
+        $xml = $imsManifestXml->saveXML();
+        $outputFilePath = realpath($this->outputPath) . '/' . $this->rawPath . '/';
+        file_put_contents($outputFilePath . '/' . $manifestFileBasename, $xml);
+    }
+
+    /**
+     * Create and zip all the files in specified folder .
+     * 
+     * @param type $contentDirPath Directory path where all the files are  
+     */
+    private function createIMSContntPackage($contentDirPath)
```

Comment:
Typo in method name (`createIMSContentPackage` instead of `createIMSContntPackage`)

Reply:
Done